Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM golang:1.10 as build
+FROM golang:1.13 as build
```

Comment:
You need to add a couple of things if you want to go to > 1.11

Please pin to 1.12 anyway

ENV GO111MODULE=off

and

ENV CGO_ENABLED=0

Cheers

Reply:
I guess you're referring to the extra stuff in the openfaas/templates repo. That's only needed because of the nested vendor folders. 

Project works fine with this docker file. Tested in a live environment.

Reply:
> I guess you're referring to the extra stuff in the openfaas/templates repo. That's only needed because of the nested vendor folders.

I am not sure what you are referring to.

Reply:
This is needed for arm platforms